solr/core/src/java/org/apache/solr/handler/component/SuggestComponent.java

```diff
@@ -519,6 +518,12 @@ private SuggesterResult toSuggesterResult(
     return result;
   }
 
+  @Override
+  public void close() throws IOException {
+    super.close();
+    IOUtils.closeQuietly(toClose);
```

Comment:
nit: normally you close down yourself first, then what's beneath you.  Reverse order of construction.